repose-aggregator/components/filters/saml-policy-translation-filter/build.gradle

```diff
@@ -32,3 +32,34 @@ dependencies {
     testCompile 'org.opensaml:opensaml-security-api'
     testCompile 'org.opensaml:opensaml-xmlsec-api'
 }
+
+license {
```

Comment:
what is this all about?

Reply:
You know how Scoverage has to instrument and compile all the things? Well, to do so, it has defined its own set of tasks which mirror the tasks added by the Java plugin (e.g., `testScoverage`, `checkScoverage`). To coordinate all of that, Scoverage also defined its own source directory set, which you've already seen. Well, the output directories in that source directory set are not the same as the output directories for, say, the java source directory set. So instead of copying resources to `${project.buildDir}/resources/test/`, the Scoverage task will copy them to `${project.buildDir}/resources/testScoverage/`.

All of this is coordination to make sure things end up where the tests run during `testScoverage` can use them, and to make sure that the test itself knows where to look. Before this change, the test itself always tried to use a hardcoded path (which matched the java output directory, but not the scoverage output directory) as the `configRoot` (i.e., the place where some test resources live).

Reply:
To simplify this a bit, I could try and set the output resources directory for the Scoverage source directory set to the same as the one for the Java source directory set... I'm not sure what consequences there are to doing so, however.